docs/knowledgebase/getting-started/glossary.md

```diff
@@ -106,75 +133,89 @@ is imperfect information about whether a component has failed) in systems that r
+each other, can come to consensus over some computation. Often mentioned alongside "**safety**" (the
+ability to ensure that any progression will eventually be agreed as having happened by all honest
+nodes) and "**liveness**" (the ability to keep making progress). Many consensus algorithm are
+referred to as being [Byzantine-fault-tolerant](#byzantine-fault-tolerance-bft).
 
-See
-https://medium.com/polkadot-network/grandpa-block-finality-in-polkadot-an-introduction-part-1-d08a24a021b5
+There is
+[a comprehensive blog series](https://polkadot.network/polkadot-consensus-part-1-introduction/) that
+provides a deep dive into the consensus strategies of [the Polkadot Network](#polkadot-network).
 
-## Crypto Primitives
+## Cryptographic Primitives
 
-The signature scheme and hashing algorithm. These are used for:
+A term that refers to concepts like signature schemes and hashing algorithms. Cryptographic
+primitives are essential to many aspects of the Substrate ecosystem:
 
-- The blockchain: blocks must be hashed under some algorithm and reference their parent block's
-  hash.
-- State: the storage is encoded as a trie allowing a cryptographic reference to any given state of
-  it, which uses a hashing algorithm.
-- Consensus: authorities must often use digital signature schemes of some kind.
-- Transaction authentication: in runtimes where transactions and accounts are used, accounts must be
-  associated with some digital identity.
-
-In the SRML, SR25519 and ED25519 is favoured over ECDSA/SECP256k1.
+- [Blockchains](#blockchain): [blocks](#block) must be hashed under some algorithm and reference
+  their parent block's hash.
+- [State](#state): Substrate storage is encoded as a [trie](#trie-patricia-merkle-tree), a data
+  structure that uses hashing to facilitate efficient verification.
+- [Consensus](#consensus): [authorities](#authority) often use digital signature schemes of some
+  kind.
+- [Transaction](#transaction) authentication: in [runtimes](#runtime) where transactions and
+  [accounts](../learn-substrate/account-abstractions) are used, accounts are identified and
+  authenticated through the use of cryptography.
 
 ## Council
 
-The council is one of the SRML governance primitives and is a body of delegates chosen from a number
-of approval votes for fixed terms. The council primarily serves as a body to optimise and
+The term "Council" is used on a number of Substrate-based networks, such as [Kusama](#kusama) and
+[Polkadot](#polkadot) to refer to an instance of
+[the Collective pallet](../runtime/frame#collective) that is used as a part of the network's
+[FRAME](#frame)-based [runtime](#runtime). The Council primarily serves as a body to optimize and
 check/balance the more inclusive referendum system. It has a number of powers:
 
-- introduce referenda with a removed or inverted (when unanimous only) AQB; and
+- introduce referenda with a removed or inverted (when unanimous only)
+  [AQB](#adaptive-quorum-biasing-aqb); and
```

Comment:
This responsability of the council is Polkadot specific. I recommend Deleting the last sentence entirely, or if you like to keep it, start it with "In Polkadot, the council is..."